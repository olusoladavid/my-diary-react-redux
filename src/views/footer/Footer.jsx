import React from "react";
import { Link } from "react-router-dom";

const Footer = () => {
  return (
    <footer className="footer">
      <div className="container">
        <h4>
          {" "}
          Built with ‚ù§ by
          <Link className="link" to="https://twitter.com/olusola_dev">
            {" "}
            Olusola
          </Link>
        </h4>
      </div>
    </footer>
  );
};

export default Footer;
