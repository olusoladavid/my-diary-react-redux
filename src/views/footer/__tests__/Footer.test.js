import Footer from "../Footer.jsx";

it("should render without crashing", () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
