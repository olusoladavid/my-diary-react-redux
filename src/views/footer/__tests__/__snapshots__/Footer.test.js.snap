// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <h4>
       
      Built with ‚ù§ by
      <Link
        className="link"
        replace={false}
        to="https://twitter.com/olusola_dev"
      >
         
        Olusola
      </Link>
    </h4>
  </div>
</footer>
`;
